microsoft-edge/devtools-guide-chromium/customize/index.md

```diff
@@ -29,9 +29,10 @@ This page lists the ways to customize Microsoft Edge DevTools.
 
 **Settings** > **Preferences** contains many options for customizing DevTools.  
 
```

Comment:
Document the button:
Settings > Preferences > Restore defaults and refresh

Reply:
Done